flow/layers/backdrop_filter_layer_unittests.cc

```diff
@@ -20,6 +21,7 @@ using BackdropFilterLayerTest = LayerTest;
 TEST_F(BackdropFilterLayerTest, PaintingEmptyLayerDies) {
   auto layer = std::make_shared<BackdropFilterLayer>(sk_sp<SkImageFilter>());
 
+  preroll_context()->cull_rect = kEmptyRect;
```

Comment:
Setting the cull rect here to an empty rect changes the nature of this test (mostly theoretically). It is supposed to be testing an empty layer with no other circumstances, but having an empty cull rect while it is operating could potentially trigger code that special cases that condition. It would be better to leave the rect at its default and change the EXPECT code than to install an empty rect to make the existing comparison correct.